### PR TITLE
add contributing guidelines

### PR DESCRIPTION
**Pull request by ghost**
adds the initial version of some contributing guidelines. not sure how we want to define the process of adding new members (via team-internal vote, or by self-promotion?), so discussing that would also be nice

/cc @nodejs/website-redesign

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+This is an adapted version of the contributing guide used by the Community Committee: https://github.com/nodejs/community-committee/blob/master/CONTRIBUTING.md
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an issue or pull request,
+  or contributing in some other way.
+* A **Collaborator** is a contributor who has been given write access to the repository.
```
Comment (bnb) [inline CONTRIBUTING.md:7]: Is it worth mentioning that they're a part of the website-redesign team?

**Review by bnb — APPROVED** [review]
LGTM, just one minor comment that isn't a blocker ❤️

File: CONTRIBUTING.md (hunk 1 above)
Comment (ghost) [reply CONTRIBUTING.md:7]: i suppose so, since they're intended to overlap

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Contributions
+
+Any change to resources in this repository must be through pull requests. This applies to all changes
+to documentation, code, binary files, etc. 
+No pull request can be merged without being reviewed.
+
+The default for each contribution is that it is accepted once no collaborator has an objection.
+During review collaborators may also request that a specific contributor who is most versed in a
+particular area gives a "LGTM" before the PR can be merged. There is no additional "sign off"
```
Comment (jestho) [inline CONTRIBUTING.md:17]: Are we not using the github review tools? 🙃

Comment (ghost) [reply CONTRIBUTING.md:17]: hmm, this integrates with the github review tools. not sure what you mean?

Comment (jestho) [reply CONTRIBUTING.md:17]: Sorry, just being a little (very?) nitpicky. I read this as the PR has to be approved with an LGTM-comment, and using the review tools is not sufficient.

Comment (ghost) [reply CONTRIBUTING.md:17]: oh, i'm fairly sure that the review tools are sufficient for giving a LGTM most of the time

**Pull request merged**